env.d/development.dist

```diff
@@ -66,3 +66,10 @@ CROWDIN_BASE_PATH=/app/src
 
 # REMINDERS
 REMINDERS_SECRET="Marsha"
+
+## POTSIE
+GRAFANA_XAPI_FUN_API_KEY=grafana_api_key
+GRAFANA_XAPI_FUN_API_ENDPOINT=https://grafana.tld/api
+GRAFANA_XAPI_FUN_API_DATASOURCE_ID=1
+GRAFANA_XAPI_FUN_API_DATASTREAM=statements-ds-marsha
+DJANGO_STAT_BACKEND=marsha.core.stats.grafana_xapi_fun_backend
```

Comment:
By default this should be not set and the dummy backend should be used. 

But the documentation (`docs/env.md`) should be updated with this.